src/panels/config/devices/device-detail/ha-device-info-card.ts

```diff
@@ -106,13 +107,31 @@ export class HaDeviceCard extends LitElement {
                 </div>
               `
             : ""}
+          ${this._getAddresses().map(
+            (connection) => html`
+              <div class="extra-info">
+                ${titleCase(connection[0])}: ${connection[1].toUpperCase()}
+              </div>
+            `
+          )}
           <slot></slot>
         </div>
         <slot name="actions"></slot>
       </ha-card>
     `;
   }
 
+  protected _getAddresses() {
+    const connections = [
+      ...this.device.connections,
+      ...this.device.identifiers,
```

Comment:
We should not use the identifiers for this.
This is connection information.

Reply:
Hmm, but then we only have the MAC addresses and not the Bluetooth addresses.

Reply:
Nope, connection info should also hold the BT address. If an integration doesn't do that, it needs fixing.

ref:
![CleanShot 2023-10-25 at 09 47 12@2x](https://github.com/home-assistant/frontend/assets/195327/249c39f1-9150-43d9-8fdf-98642908bcaf)

Reply:
Okay, then I picked the ones that don't do it right :D